manifests/d/dorssel/usbipd-win/2.0.0/dorssel.usbipd-win.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: dorssel.usbipd-win
+PackageVersion: 2.0.0
+PackageLocale: en-US
+Publisher: Frans van Dorsselaer
+PublisherUrl: https://github.com/dorssel
+PublisherSupportUrl: https://github.com/dorssel/usbipd-win/issues
+# PrivacyUrl: 
+# Author: 
+PackageName: usbipd-win
+PackageUrl: https://github.com/dorssel/usbipd-win
+License: GNU General Public License, version 2
+LicenseUrl: https://raw.githubusercontent.com/dorssel/usbipd-win/master/COPYING.md
```

Comment:
I'll accept this change. Originally this was a URL that pointed to the license *for this specific version of the software*. Given that this software is GPL2.0-only, which is sticky and thus permanent, it doesn't really matter. But please consider allowing a version-dependent LicenseUrl. For other software, the license in the master branch may not represent the license for this specific version of the installer.

Reply:
Okay